package.json

```diff
@@ -25,6 +25,7 @@
     "commit": "git-cz",
     "lint": "sl-scripts lint",
     "lint.fix": "yarn lint --fix",
+    "push": "yarn build && cd dist && yalc push",
```

Comment:
To be removed or renamed?

Reply:
actually I really love this. I think it should be included in `sl-scripts`, wondered what others thought?

Reply:
I would call it `publish.yalc` or so. Push is very similar to `git push`

Reply:
I like it, I do it all the time. `push.yalc`?

Reply:
Yeah, `push.yalc` looks way better.
I agree it could be useful - actually, I do it all the time as well.

Reply:
```suggestion
    "yalc.push": "yarn build && cd dist && yalc push",
```
:)